yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java

```diff
@@ -7,34 +7,39 @@
  */
 public interface HumanProfile extends Profile {
 
+    String GENDER_MALE = "MALE";
```

Comment:
I think you were planning to have these as constants.
If so, please update accordingly.

Reply:
That is a constant.  _public final_ is redundant, because its being defined on an interface (public + final by default)

Reply:
sorry, hadn't seen it was an interface, my bad :-(